src/components/NavBar/NavBar.tsx

```diff
@@ -1,11 +1,37 @@
+import { useState } from 'react';
 import { NavItem } from '../../types/navBar';
+import NavBarItem from '../NavBarItem/NavBarItem';
+import styles from './navBar.module.css';
 
 interface NavBarProps {
   items: NavItem[];
 }
 
-const NavBar = (_: NavBarProps) => {
-  return <></>;
+const NavBar = ({ items }: NavBarProps) => {
+  const [navItems, setNavItems] = useState(items);
```

Comment:
This is not good, just simply pass current path where user is.
Ex: If current user is on the product pass, then we pass `/product` to this component then we render which item is match